### PR TITLE
Updated EKS blog draft

### PR DESCRIPTION
**Pull request by infin8x**
Some tweaks to Levi's EKS post

### CODE REVIEW

File: content/blog/multi-lang-components/index.md [hunk 1]
```diff
@@ -14,29 +14,31 @@ tags:
     - typescript
 ---
 
-Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
-first multi-language component is the popular [EKS package], which was previously only available for TypeScript. With the
-[v0.22.0 release], teams can [manage EKS clusters] in their language of choice (Python, .NET, Go, and TypeScript).
+Pulumi's infrastructure as code tooling combines the programming languages and tools you already know with the full power of cloud
+infrastructure. But, until now, some of that Pulumi components for that cloud infrastructure, like our popular [EKS package], were
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:18]: infrastructure. But, until now, some  Pulumi components for cloud infrastructure, like our popular [EKS package], were

File: content/blog/multi-lang-components/index.md [hunk 2]
```diff
@@ -14,29 +14,31 @@ tags:
...
+only available in a subset of the languages supported by Pulumi.
 
-<!--more-->
+Now, with multi-language components, anyone can author a Pulumi component once, in the language they prefer, and make it available
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:21]: Now, with multi-language components, anyone can author a Pulumi component, in the language they prefer, and make it available

File: content/blog/multi-lang-components/index.md [hunk 3]
```diff
@@ -14,29 +14,31 @@ tags:
...
+across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:22]: is there supposed to be a link to the package?

File: content/blog/multi-lang-components/index.md [hunk 4]
```diff
@@ -14,29 +14,31 @@ tags:
...
+and brought it to all four Pulumi languages: TypeScript, Python, .NET, and Go. Now, regardless of the language you choose, you can
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:23]: and made it available to all four Pulumi languages: TypeScript, Python, .NET, and Go. Regardless of the language you choose, you can

File: content/blog/multi-lang-components/index.md [hunk 5]
```diff
@@ -14,29 +14,31 @@ tags:
...
-language works best for each team. It's already possible to collaborate across languages by using separate Pulumi projects
-and referring to them using [Stack References]. Multi-language components take this one step further towards the vision of
-"write once, use anywhere". Component authors broaden their reach across languages, and users benefit from the work
-of a much larger community.
+{{< youtube "gxLyAr0lUg0?rel=0" >}}
 
-Multi-language component support allows component authors to project their API into all Pulumi-supported languages
-idiomatically and without requiring knowledge of all of these languages. First-class authoring support for multi-language
-components is in the works.
+<!--more-->
```
Comment (spara) [inline content/blog/multi-lang-components/index.md:29]: is this cut line necessary?

**Review by spara — CHANGES_REQUESTED** [review]
Really like the point about sharing - adds to virility of component authoring.

File: content/blog/multi-lang-components/index.md (hunk 3 above)
Comment (infin8x) [reply content/blog/multi-lang-components/index.md:22]: Yeah - I think it's covered at the bottom? Borrowed Levi's existing `[EKS package]` reference.

File: content/blog/multi-lang-components/index.md (hunk 5 above)
Comment (infin8x) [reply content/blog/multi-lang-components/index.md:29]: I figured we had to have one for the blog to render well - will defer to you on what's best here.

Comment (spara) [reply content/blog/multi-lang-components/index.md:29]: I'll merge it and see how it renders in Hugo

File: content/blog/multi-lang-components/index.md (hunk 3 above)
Comment (spara) [reply content/blog/multi-lang-components/index.md:22]: ah, yes

**Pull request merged**